openapi/openapi/src/main/scala/endpoints/openapi/model/OpenApi.scala

```diff
@@ -308,12 +344,45 @@ object OpenApi {
           "info" -> infoJson(openApi.info),
           "paths" -> pathsJson(openApi.paths)
         )
+      if (openApi.tags.nonEmpty) {
+        fields += "tags" -> new ujson.Arr(
+          openApi.tags.map(tag => tagJson(tag)).to(mutable.ArrayBuffer)
```

Comment:
This doesn’t compile on 2.12. Can you see if importing `scala.collection.compat._` fixes the issue?

Reply:
When I add that it seems it is not used. I could also go back to the constructor as before (`tags: _*`)

Reply:
Yeah, that’s also a solution :)

It’s unused only on 2.13, though, so you should be able to remove the warning with a `@nowarn` annotation (I believe). But your alternative solution might be simpler :)